frontend/src/scenes/web-analytics/tiles/WebAnalyticsTile.tsx

```diff
@@ -121,6 +124,21 @@ const BreakdownValueCell: QueryContextColumnComponent = (props) => {
                 )
             }
             break
+        case WebStatsBreakdown.Language:
+            if (typeof value === 'string') {
+                const [language, countryCode] = value.split('-')
+
+                // Locales are complicated, the country code might be hidden in the second part
+                // of the locale
+                const parsedCountryCode = countryCode?.match(/([A-Z]{2})/)?.[0] ?? ''
+                return (
+                    <>
+                        {countryCodeToFlag(parsedCountryCode)}&nbsp;
```

Comment:
Let's add a `fallbackFlag(language)`, it doesn't need many entries, but would have `nl`

Reply:
I've only added `nl` because it's probably the most impactful one that hasn't got any followups (Germany would be too, but there's `de-DE` and `de-AT`, and Poland seems to always be `pl-PL`)

Reply:
Once we have this running in prod I can go back and check if there's any obvious offender here